client/components/Byline/Byline.tsx

```diff
@@ -3,27 +3,23 @@ import React from 'react';
 import ensureUserForAttribution from 'utils/ensureUserForAttribution';
 import { joinOxford, naivePluralize } from 'utils/strings';
 
-type OwnProps = {
+export type Props = {
```

Comment:
I've been renaming these like `ComponentNameProps`, so this would become `BylineProps`. Do you prefer just `Props`?

Reply:
That makes sense to me, I can change it!